lib/master_proxy/master_proxy.ex

```diff
@@ -49,22 +49,6 @@ defmodule MasterProxy do
     Supervisor.start_link(children, supervisor_opts)
   end
 
-  defp merge_runtime_and_compiled_opts(options) do
-    relevant_keys = [:http, :https, :log_requests, :conn, :backends, :name]
-    compiled_opts = Application.get_all_env(:master_proxy)
-
-    env_configed =
-      relevant_keys
-      |> Enum.reduce([], fn key, acc ->
-        set_opt_or_skip(key, acc, compiled_opts)
-      end)
-
-    relevant_keys
-    |> Enum.reduce(env_configed, fn key, acc ->
-      set_opt_or_skip(key, acc, options)
-    end)
-  end
-
   def set_opt_or_skip(key, target_list, source_list) do
```

Comment:
I think this function is unused now.

Also, most idiomatic would probably be moving this file up one directory (lib/master_proxy.ex). What are your thoughts on that?

Reply:
Ah right, and that seems fine to me.